index.js

```diff
@@ -99,13 +103,37 @@ var pcbStackup = function(layers, options, done) {
       layerOptions.plotAsOutline = options.outlineGapFill
     }
 
-    var converter = gerberToSvg(layer.gerber, layerOptions, finishLayer)
+    var converterDone = function() {
+      layer.done = true
+    }
+
+    var converter
+
+    if (layer.converter != null && layer.gerber == null) {
+      converter = layer.converter
+      converterDone()
+    }
+    else {
+      converter = gerberToSvg(layer.gerber, layerOptions, converterDone)
+    }
 
     stackupLayers.push({
       type: layerType,
       converter: converter,
       options: layerOptions
     })
+
+    var finishIfDone = function() {
```

Comment:
Ahrrg, yeah, yours is better.  The only thing different about is with mine you could tack on a new gerber and it would take care of it. I guess you have to remove the converter in that case. Not sure of the use-case anyway.